CHANGELOG.md

```diff
@@ -104,6 +104,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   and `jj split any-non-existent-path` inserts an empty commit between the
   target commit and its parents.
 
+* jj split gained new `--empty-parent` and `--empty-child` options. They
+  allow inserting an empty parent before or after an empty child after the
```

Comment:
This needs a comma and has too many "after"s. "an empty parent before, or an empty child after..."

Reply:
Done